ibm/service/atracker/resource_ibm_atracker_settings.go

```diff
@@ -49,13 +49,12 @@ func ResourceIBMAtrackerSettings() *schema.Resource {
 				Description: "If present then only these regions may be used to define a target.",
 				Elem:        &schema.Schema{Type: schema.TypeString},
 			},
-			// Future Planned support
-			// "metadata_region_backup": &schema.Schema{
-			// 	Type:         schema.TypeString,
-			// 	Optional:     true,
-			// 	ValidateFunc: validate.InvokeValidator("ibm_atracker_settings", "metadata_region_backup"),
-			// 	Description:  "Provide a back up region to store meta data.",
-			// },
+			"metadata_region_backup": {
+				Type:         schema.TypeString,
```

Comment:
if user doesn't provide metadata_region_backup will we have any default value?
In Read we are setting back
```
d.Set("metadata_region_backup", settings.MetadataRegionBackup)
```
what value we get?
if we have some value we should mark it as both optional and computed

Reply:
There is no default value for this, it is expected to return `""` if nothing has been set.